docs/resources/policy_transit_gateway_static_route.md

```diff
@@ -0,0 +1,79 @@
+---
+subcategory: "beta"
+page_title: "NSXT: nsxt_policy_transit_gateway_static_route"
+description: A resource to configure a static route
+---
+
+# nsxt_policy_transit_gateway_static_route
+
+This resource provides a method for the management of a transit gateway StaticRoute.
+
+This resource is applicable to NSX Global Manager, NSX Policy Manager and VMC.
+
+## Example Usage
+
+```hcl
+data "nsxt_policy_project" "proj" {
+  display_name = "demoproj"
+}
+
+data "nsxt_policy_transit_gateway" "tgw1" {
+  context {
+    project_id = data.nsxt_policy_project.proj.id
+  }
+  display_name = "TGW1"
+}
+
+resource "nsxt_policy_transit_gateway_static_route" "test" {
+  display_name         = "test"
+  description          = "Terraform provisioned StaticRoute for transit gateway"
+  parent_path          = data.nsxt_policy_transit_gateway.tgw1.path
+  enabled_on_secondary = false
+  network              = "3.3.3.0/24"
+  next_hop {
+    ip_address     = "192.168.1.1"
+    admin_distance = 10
+    scope          = ["/orgs/default/projects/<project-id>/transit-gateways/default/attachments/<attachment-id>"]
```

Comment:
We usually do not put hardcoded paths in example code, and never have placeholders (e.g <project-id>).
Normally we use a reference as you do here for nsxt_policy_project and nsxt_policy_transit_gateway - either with a data source or a resource.
Basically it's similar to what you have in the acceptance test.